### DISCUSSION TITLE
Allow hiding "Used Plugins" table

### DISCUSSION THREAD

**Discussion by alexking** [fastlane/fastlane#19484] [Ideas] [answered]
Allow users to hide the `Used Plugins` table that shows whenever `fastlane` is run - it can get quite long and isn't always needed. Would be happy to implement this if there's interest, imagine we could model it after `FASTLANE_SKIP_UPDATE_CHECK`, and hook into [this `skip_print_plugin_info` check](https://github.com/fastlane/fastlane/blob/e874a47c6e2e0e61590a03d3b71e75e5a505d1ce/fastlane/lib/fastlane/plugins/plugin_manager.rb#L313).

**Comment by iBotPeaches** [comment, marked as answer]
I googled this exact thing - found this thread. Sad it wasn't resolved, then learned it was.

```
FASTLANE_HIDE_PLUGINS_TABLE=true
```

So I'll close this 2021 issue out
